com.unity.toonshader/Documentation~/FeatureModel_en.md

```diff
@@ -1,100 +1,105 @@
-| Use the alpha channel of Emissive Map as a Clipping mask 	| OK 	| OK 	| OK 	|  	|
-| Emission Map Animation	| OK 	| OK 	| OK 	|  	|
-| Base Speed (Time)	| OK 	| OK 	| OK 	|  	|
-| Animation Mode	| OK 	| OK 	| OK 	|  	|
-| Scroll U/X direction 	| OK 	| OK 	| OK 	|  	|
-| Scroll V/Y direction	| OK 	| OK 	| OK 	|  	|
-| Rotate around UV center 	| OK 	| OK 	| OK 	|  	|
-| Ping-pong moves for base 	| OK 	| OK 	| OK 	|  	|
-| Color Shifting with Time 	| OK 	| OK 	| OK 	|  	|
-| Color Shifting with View Angle	| OK 	| OK 	| OK 	|  	|
-| ***9. Angel Ring Projection Settings*** 	|  	|  	|  	|  	|
-| Angel Ring	| OK 	| OK 	| OK 	|  	|
-| Offset U/V	| OK 	| OK 	| OK 	|  	|
-| Alpha Channel as Clipping Mask	| OK 	| OK 	| OK 	|  	|
-| ***10. Scene Light Effectiveness Settings*** 	|  	|  	|  	|  	|
-Activate/Deactive LightColor Effectiveness to each color:<br> Base Color,1st Shading Color, 2nd Shading Color, Highlight, Rim Light, Inverted Light Direction Rim Light, MatCap, Angel Ring, or Outline 	| OK 	| OK 	| OK 	|  	|
-| ***11. Metaverse Settings*** 	|  	|  	|  	|  	|
-| Metaverse Light 	| OK 	| OK 	| N.A. 	|  	|
-| Metaverse Light Intensity	| OK 	| OK 	| N.A. 	|  	|
-| Metaverse Light Direction	| OK 	| OK 	| N.A. 	|  	|
-| ***12. Outline Settings*** 	|  	|  	|  	|  	|
-| Outline Mode 	| OK 	| OK 	| OK 	|  	|
-| Outline Width 	| OK 	| OK 	| OK 	|  	|
-| Outline Color	| OK 	| OK 	| OK 	|  	|
-| Blend Base Color to Outline	| OK 	| OK 	| OK 	|  	|
-| Offset Outline with Camera Z-axis	| OK 	| OK 	| OK 	|  	|
-| Camera Distance for Outline Width 	| OK 	| OK 	| OK 	|  	|
-| Outline Color Map	| OK 	| OK 	| OK 	|  	|
-| Rotate around UV center 	| OK 	| OK 	| OK 	|  	|
-| Ping-pong moves for base 	| OK 	| OK 	| OK 	|  	|
-| Color Shifting with Time 	| OK 	| OK 	| OK 	|  	|
-| Color Shifting with View Angle	| OK 	| OK 	| OK 	|  	|
-| ***13.Tessellation Settings*** 	|  	|  	|  	|  	|
-| Tessellation Settings 	| DX11/Vulkan/Metal 	| N.A. 	|  DX11/Vulkan/Metal 	|  	|
-| ***14. EV Adjustment*** 	|  	|  	|  	|  	|
-| EV Adjustment in high intensity light scenes	 	| N.A	| N.A.	| OK	|  	|
-| ***15. Render pipeline built-in ray-traced shadows*** 	|  	|  	|  	|  	|
-| DXR shadow supported in render pipelines 	| N.A.	| N.A.	| OK	|  	|
-| ***16. Box Light*** 	|  	|  	|  	|  	|
-| Substitute for directional light 	| N.A.	| N.A.	| OK	| To get around the limitation that multiple directional lights can't cast shadows at the same time. |
+| Function	                                                                                                                                                                                                 | Legacy 	            | URP 	  | HDRP 	              | Note 	|
+|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------|--------|---------------------|-	|
```

Comment:
I observe we started to use  Built in RP  in the  , so we might have to rename Legacy to BiRP, or Built in RP, But should it be in another PR?

Reply:
Let's do it in another PR to "refactor" this doc